src/fontcache.h

```diff
@@ -227,11 +227,9 @@ inline FontCacheSubSetting *GetFontCacheSubSetting(FontSize fs)
 std::string GetFontCacheFontName(FontSize fs);
 
 void DebugPrintFontSettings(const std::string& desc);
-
-void InitFontCache(bool monospace);
-void UninitFontCache();
-
 bool GetFontAAState();
+void InitFontCache(bool monospace);
 void SetFont(FontSize fontsize, const std::string &font, uint size);
+void UninitFontCache();
```

Comment:
Why shuffle these?

Reply:
I alphabetized them. Is there a consistent scheme for ordering things like this? I don't see anything mentioned in the style guide.

Reply:
They're usually grouped for a reason Init/Uninit being related. Don't random alphabetize things.